appveyor.yml

```diff
@@ -8,17 +8,17 @@ environment:
 
   matrix:
 
-    # FIXME: https://ci.appveyor.com/project/isaachier/hunter/build/1.0.476
+    # FIXME: https://ci.appveyor.com/project/JochenGuck/hunter/builds/25660219/job/efu4x48fg3jiif4j
```

Comment:
It's not clear what is the intention. Which toolchain you're trying to add/remove?

Reply:
I used the same tool chains as in thrift version 0.11 but updated the reports of the fail of the not supported tool chains

Reply:
> updated the reports of the fail of the not supported tool chains

It was not necessary. The essential part is removing/adding toolchains.

Reply:
Ok sorry, than the original appveyor.yml  will also do it